pyanaconda/core/glib.py

```diff
@@ -0,0 +1,61 @@
+# Wrappers for GLib functions.
+#
+# Use these only if there is no other way.
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+#  Author(s):  Jiri Konecny <jkonecny@redhat.com>
+#
+
+import gi
+gi.require_version("GLib", "2.0")
+
+from gi.repository.GLib import markup_escape_text, format_size_full, \
```

Comment:
You can use a wild card here, since there is defined `__all__`.

Reply:
Yep, it could be nice but we have this in our guidelines:
```
Never do wildcard imports with the exception of pyanaconda.dbus.typing
```
So changing the guidelines?

Reply:
I think that this is a valid use case for an exception, so yes.